.github/workflows/ruby-dataset-measure.yml

```diff
@@ -40,11 +47,13 @@ jobs:
           path: ${{ github.workspace }}/repo
       - name: Create database
         run: |
-          codeql database create \
+          codeql database init \
             --search-path "${{ github.workspace }}/ruby/extractor-pack" \
-            --threads 4 \
             --language ruby --source-root "${{ github.workspace }}/repo" \
             "${{ runner.temp }}/database"
+          codeql database trace-command --threads 4 --index-traceless-dbs "${{ runner.temp }}/database"
+          codeql database index-files -l xml --include "**/*.xml.erb" "${{ runner.temp }}/database"
```

Comment:
I am a bit surprised that our xml parser handles `xml.erb` files in the general case. Do you think there could be spurious syntax errors, or will it be guaranteed to work?

Reply:
> That's a good point. I don't think it would work in the general case, but it might be alright for all the .xml.erb files in the five projects above.

There are plenty of XML syntax errors in the `xml.erb` files in the 5 projects we currently use. However, these errors are non-fatal. The purpose is just to gather some stats about XML-like data. For that purpose it works fine.